e2e/stress/IoTClientPerf/readme.md

```diff
@@ -0,0 +1,58 @@
+# Azure IoT Hub Stress Test
+
+Use `dotnet run` to display the help page for this tool.
+
+## Prerequisites
+
+##### Create an IoT Hub for performance testing.
+##### Create one IoT Hub device with SAS token and one with X509 certificate authentication.
+
+Fill in the following environment variables (see the e2e\test prerequisites for details):
+```
+IOTHUB_CONN_STRING_CSHARP=
+IOTHUB_DEVICE_CONN_STRING=
```

Comment:
I had removed these (IOTHUB_DEVICE_CONN_STRING, IOTHUB_DEVICE_CONN_STRING2) from our pre-requisites, since the E2E tests were no longer referencing them. We will need to add them back in if they are needed by the stress test.

Reply:
These are needed for samples as well. We need them back.